actions/slack-notify-git-ref/payload.json

```diff
@@ -1,27 +1,27 @@
 {
-  "text": "New git ref for ${{ env.GITHUB_REPO }}",
+  "text": "New git ref for <https://github.com/${GITHUB_REPOSITORY}|${GITHUB_REPOSITORY}>",
```

Comment:
hmm why is GITHUB_REPOSITORY referenced twice?

Reply:
The second one is the anchor text.